Find a functionally equivalent code:android.graphics.Canvas.setMatrix:COMMENT
Method Modifier: public      
Comment:/**
 * Completely replace the current matrix with the specified matrix. If the
 * matrix parameter is null, then the current matrix is reset to identity.
 *
 * <strong>Note:</strong> it is recommended to use {@link #concat(Matrix)},
 * {@link #scale(float, float)}, {@link #translate(float, float)} and
 * {@link #rotate(float)} instead of this method.
 *
 * @param matrix The matrix to replace the current matrix with. If it is
 * null, set the current matrix to identity.
 *
 * @see #concat(Matrix)
 */

@@ -1,3 +1,3 @@
 {
-    native_setMatrix(mNativeCanvas, matrix == null ? 0 : matrix.native_instance);
+    native_setMatrix(mNativeCanvasWrapper, matrix == null ? 0 : matrix.native_instance);
 }

